synnax/cmd/start.go

```diff
@@ -80,11 +80,11 @@ func start(cmd *cobra.Command) {
 	v := version.Get()
 	decodedName, _ := base64.StdEncoding.DecodeString("bGljZW5zZS1rZXk=")
 	var (
-		ins      = configureInstrumentation(v)
-		insecure = viper.GetBool("insecure")
-		verbose  = viper.GetBool("verbose")
-		autoCert = viper.GetBool("auto-cert")
-		verifier = viper.GetString(string(decodedName))
+		ins, prettyLogger = configureInstrumentation(v)
```

Comment:
why do we need a separate logger and can't just use `ins` to log the start & shutdown messages?

Reply:
Because the production logger sometimes won't let us print nice output